src/nvim/CMakeLists.txt

```diff
@@ -189,8 +192,14 @@ list(APPEND NVIM_LINK_LIBRARIES
     ${LIBUV_LIBRARIES}
     ${MSGPACK_LIBRARIES}
     ${LUAJIT_LIBRARIES}
+    "-Wl,--whole-archive"
```

Comment:
I'm trying to understand why the change in order was necessary. Is it because unibilium/termkey are skipped when it determines none of its symbols are directly used by the nvim binary? If so then it actually makes sense.